nemo_skills/inference/generate.py

```diff
@@ -147,6 +148,9 @@ def _post_init_validate_server(self):
 
     def _post_init_validate_params(self):
         """Validate that certain parameters are restricted to certain values"""
+        if self.prompt_format not in ["ns", "openai"]:
```

Comment:
Please add assert that prompt_config and prompt_template are both set to None if format != ns